src/services/recommendations.ts

```diff
@@ -1,7 +1,8 @@
 import { Events } from '@storefront/flux-capacitor';
-import { BaseService } from '../core/service';
+import { core, BaseService } from '../core/service';
 import StoreFront from '../storefront';
 
+@core
```

Comment:
do we have to make the recommendations service a core service as part of this work?